{"pageProps":{"page":{"slug":"exception.getline.md","content":"# Exception::getLine\n\n(PHP 5, PHP 7, PHP 8)\n\nException::getLine — Отримує рядок, у якому виник виняток\n\n### Опис\n\n```methodsynopsis\nfinal public Exception::getLine(): int\n```\n\nОтримати номер рядка, де виняток було створено\n\n### Список параметрів\n\nЦя функція не має параметрів.\n\n### Значення, що повертаються\n\nПовертає номер рядка, де було створено виняток.\n\n### Приклади\n\n**Приклад #1 Приклад використання** Exception::getLine()\\*\\*\\*\\*\n\n```php\n<?php\ntry {\n    throw new Exception(\"Какое-нибудь сообщение об ошибке\");\n} catch(Exception $e) {\n    echo \"Исключение было создано на строке: \" . $e->getLine();\n}\n?>\n```\n\nВисновок наведеного прикладу буде схожим на:\n\n```\nИсключение было создано на строке: 3\n```\n\n### Дивіться також\n\n-   [Throwable::getLine()](throwable.getline.md) \\- Отримує рядок скрипта, в якому цей об'єкт був викинутий\n","title":"Exception::getLine","contentType":2,"navigation":[{"exception.getfile.md":"« Exception::getFile"},{"exception.gettrace.md":"Exception::getTrace »"},{"index.md":"PHP Manual"},{"class.exception.md":"Exception"}]}},"__N_SSG":true}